### PR TITLE
Fixing empty volumes list when calling for all droplets

### PR DESCRIPTION
**Pull request by wacuuu**
This may not be the best solution, as this is writing the same thing twice, but let's discuss.

### CODE REVIEW

**Comment by koalalorenzo** [comment]
What I would do is:

1. Create a new unit test to simulate the error reported in #281 (aka Volumes should not be empty)
2. Implement a proper solution that is using Objects rather than IDs. This might be slow or not nice so be careful!

**Comment by wacuuu** [comment]
No worries, I will definitely do the tests, but please explain. volume_ids come as field in response per droplet, so no extra requests would be done. What I would like to do is to simply volumes = response.volume_ids(more or less). Do I miss something?

**Comment by koalalorenzo** [comment]
It is more about what are the expectations based on the name of the variables:

`volumes_ids` it is 100% clear to be an array of IDs
`volumes` it is not clear to be an array of IDs, instead I would expect to be an array of [Volume](https://github.com/koalalorenzo/python-digitalocean/blob/master/digitalocean/Volume.py#L5)s

😄

**Comment by wacuuu** [comment]
Soooo... what do you think?

File: digitalocean/Droplet.py [hunk 1]
```diff
@@ -639,5 +640,15 @@ def get_kernel_available(self):
 
         return kernels
 
+    def update_volumes_data(self):
+        """
+            Trigger refresh of volumes table
```
Comment (koalalorenzo) [inline digitalocean/Droplet.py]: Add more description to just "trigger". What is it really doing? Why do we need to run this?
Ex: write more about what data is populating and what are the expected results by running it.

File: digitalocean/Droplet.py [hunk 2]
```diff
@@ -639,5 +640,15 @@ def get_kernel_available(self):
...
+        """
+        self.volumes = list()
+
+        for volume in self.volume_ids:
+            self.volumes.append(Volume().get_object(self.token, volume))
```
Comment (koalalorenzo) [inline digitalocean/Droplet.py]: can we make it a little more readable and KISS?
One line is not a style I want

File: digitalocean/Droplet.py (hunk 1 above)
Comment (wacuuu) [reply digitalocean/Droplet.py]: Done

File: digitalocean/Droplet.py (hunk 2 above)
Comment (wacuuu) [reply digitalocean/Droplet.py]: Understandable :)

**Comment by koalalorenzo** [comment]
🎊 yasss! 👍 💪 

Thank you for contributing

**Pull request merged**